internal/distributed/proxy/httpserver/handler_v1.go

```diff
@@ -946,3 +952,149 @@ func (h *HandlersV1) search(c *gin.Context) {
 		}
 	}
 }
+
+func (h *HandlersV1) import_(c *gin.Context) { //nolint:all
+	httpReq := ImportReq{
+		DbName: DefaultDbName,
+	}
+	if err := c.ShouldBindWith(&httpReq, binding.JSON); err != nil {
+		log.Warn("high level restful api, the parameter of ImportReq is incorrect", zap.Any("request", httpReq), zap.Error(err))
+		c.AbortWithStatusJSON(http.StatusOK, gin.H{
+			HTTPReturnCode:    merr.Code(merr.ErrIncorrectParameterFormat),
+			HTTPReturnMessage: merr.ErrIncorrectParameterFormat.Error() + ", error: " + err.Error(),
+		})
+		return
+	}
+	if httpReq.CollectionName == "" || len(httpReq.Files) == 0 {
+		log.Warn("high level restful api, import require parameter: [collectionName, files], but miss")
+		c.AbortWithStatusJSON(http.StatusOK, gin.H{
+			HTTPReturnCode:    merr.Code(merr.ErrMissingRequiredParameters),
+			HTTPReturnMessage: merr.ErrMissingRequiredParameters.Error() + ", required parameters: [collectionName, files]",
+		})
+		return
+	}
+	req := &internalpb.ImportRequest{
+		DbName:         httpReq.DbName,
+		CollectionName: httpReq.CollectionName,
+		PartitionName:  httpReq.PartitionName,
+		Files: lo.Map(httpReq.Files, func(paths []string, _ int) *internalpb.ImportFile {
+			return &internalpb.ImportFile{Paths: paths}
+		}),
+		Options:        funcutil.Map2KeyValuePair(httpReq.Options),
+		ClusteringInfo: httpReq.ClusteringInfo,
+	}
+	username, _ := c.Get(ContextUsername)
+	ctx := proxy.NewContextWithMetadata(c, username.(string), req.GetDbName())
+	response, err := h.executeRestRequestInterceptor(ctx, c, req, func(reqCtx context.Context, req any) (any, error) {
+		return h.proxy.ImportV2(ctx, req.(*internalpb.ImportRequest))
+	})
+	if err == RestRequestInterceptorErr {
+		return
+	}
+	if err == nil {
+		err = merr.Error(response.(*internalpb.ImportResponse).GetStatus())
+	}
+	if err != nil {
+		c.JSON(http.StatusOK, gin.H{HTTPReturnCode: merr.Code(err), HTTPReturnMessage: err.Error()})
+	} else {
+		resp := response.(*internalpb.ImportResponse)
+		c.JSON(http.StatusOK, gin.H{HTTPReturnCode: http.StatusOK, HTTPReturnImportRequestID: resp.GetRequestID()})
+	}
+}
+
+func (h *HandlersV1) getImportProgress(c *gin.Context) {
+	httpReq := GetImportProgressReq{
+		DbName: DefaultDbName,
+	}
+	if err := c.ShouldBindWith(&httpReq, binding.JSON); err != nil {
```

Comment:
Http GET method, always use `c.Query($paramName, $defaultValue)
for example: $path + "? requestID=XXX&dbName=XXX

* both getImportProgress and listImports *